astroid/raw_building.py

```diff
@@ -324,7 +326,7 @@ def _build_from_function(
         object_build_function(node, member, name)
 
 
-def _safe_has_attribute(obj, member) -> bool:
+def _safe_has_attribute(obj, member: str) -> bool:
     try:
         return hasattr(obj, member)
     except Exception:  # pylint: disable=broad-except
```

Comment:
This seems like we could easily avoid the broad exception here.. But I'm overscoping :sweat_smile:

Reply:
Is there any reason for this function to exist? Are there any objects for which `hasattr` could raise an error? My guess is that this is a holdover from Python 2 days.

Reply:
Yeah I don't think so. Let's remove, it is private anyway.